server/bundles/io.cloudbeaver.server/src/io/cloudbeaver/service/sql/WebSQLProcessor.java

```diff
@@ -203,6 +208,16 @@ public WebSQLExecuteInfo processQuery(
                         webDataFilter.getOffset(),
                         webDataFilter.getLimit()))
                     {
+                        SqlLogStateJob sqlLogStateJob = null;
+                        if (readLogs) {
+                            final DBPDataSource dataSource = context.getDataSource();
```

Comment:
```suggestion
                            DBPDataSource dataSource = context.getDataSource();
```

we don't use `final` by our checkstyle